packages/js/product-editor/src/blocks/summary/edit.tsx

```diff
@@ -65,26 +85,29 @@ export function Edit( {
 			</BlockControls>
 
 			<BaseControl
-				id={ id }
+				id={ contentId }
 				label={ label || __( 'Summary', 'woocommerce' ) }
 			>
-				<RichText
-					id={ id }
-					identifier="content"
-					tagName="p"
-					value={ summary }
-					onChange={ setSummary }
-					placeholder={ __(
-						"Summarize this product in 1-2 short sentences. We'll show it at the top of the page.",
-						'woocommerce'
-					) }
-					data-empty={ Boolean( summary ) }
-					className={ classNames( 'components-summary-control', {
-						[ `has-text-align-${ align }` ]: align,
-					} ) }
-					dir={ direction }
-					allowedFormats={ allowedFormats }
-				/>
+				<div { ...blockProps }>
```

Comment:
Do you think that moving this from the top most element is not gonna cause any rare behavior in the future?
I'm just curious about it.

Reply:
I was wondering the same.  It might cause some issues in the future, but it's pretty easy to revert if that happens.

If that turns out to be the case, we may try creating a class in the editor root to determine current block selection and position the toolbar accordingly.  This is not an ideal solution, but currently there is no way to pass any props to the toolbar popover.